pandamium_datapack/data/pandamium/functions/misc/teleport/anti_tp_exploit.mcfunction

```diff
@@ -0,0 +1,3 @@
+#before teleporting, this function should run to prevent hackers from tracking the teleport destination
+execute at @s[gamemode=!spectator] run function pandamium:misc/teleport/anti_tp_exploit_
```

Comment:
Why is it not run for players in spectator mode?

Reply:
The position of spectating players isn't sent to non-spectating players

Reply:
ah makes sense